sdk/python/feast/feature_view_projection.py

```diff
@@ -27,6 +27,7 @@ class FeatureViewProjection:
 
     name: str
     name_alias: Optional[str]
+    desired_features: List[str]
```

Comment:
Is this also a bug for the go feature server?

Reply:
Sorry, what do you mean?

Reply:
The go feature server also implements feature view projection and featureservice. Could this bug also be a bug for the go feature service retrieval?